src/service/MosaicAmountView.ts

```diff
@@ -63,6 +55,6 @@ export class MosaicAmountView {
      * @returns {string}
      */
     public fullName(): string {
```

Comment:
Do we still need this method? or perhaps change it to be more meaningful e.g. ``` public mosaicNonce() ```

Reply:
This method will probably be re-used when mosaics naming has been reintroduced. Renaming it "name()" could be good. 

Keeping the method as is allows for better backwards compatibility, people may have been using this method so I preferred to update the method to return the hexadecimal representation of the mosaic ID, this will be changed.